src/redux/store.js

```diff
@@ -0,0 +1,8 @@
+import { configureStore } from '@reduxjs/toolkit';
+// import counterReducer from '../features/counter/counterSlice';
```

Comment:
- Please, let's keep our codebase clean by removing this commented code from the codebase.

Reply:
Thanks, @zieeco for pointing this out. "ll fix it right away.